### ISSUE TITLE
fix home page to remove banner

### ISSUE THREAD

**Issue by modernNeo** [CSSS/csss-site#631]
the banner is rather big and unnecessary on the home page when viewing on a phone. It might be better to shrink it on mobile [or maybe even just remove it?]

@EarthenSky 
![Screenshot from 2023-12-16 15-34-36](https://github.com/CSSS/csss-site/assets/14242088/dd33eb27-25db-4bdc-9abc-6afbd66d8492)

**Comment by EarthenSky** [comment]
Good point! I would like to fix this as well. I'll look into it soonish
